modules/dns/dns.rb

```diff
@@ -1,20 +1 @@
 require 'dns/dns_plugin'
-
-module Proxy::Dns
```

Comment:
OK, fine by me if you want to change it - then my corresponding comment in the DHCP PR is all the more important, let's make sure we get it correct this second time.